dkg/pedersen/verifiableEncDec_test.go

```diff
@@ -0,0 +1,202 @@
+package pedersen
```

Comment:
Outside this PR: the functions in this file are quite long and complex, refactoring into smaller functions for readability might be helpful